libs/components/modals/src/lib/modules/modal/modal.service.ts

```diff
@@ -42,20 +47,26 @@ export class SkyModalService {
     config?: SkyModalConfigurationInterface | any[]
   ): SkyModalInstance {
     const modalInstance = new SkyModalInstance();
-    this.createHostComponent();
-    const params = this.getConfigFromParameter(config);
+    this.#createHostComponent();
+    const params = this.#getConfigFromParameter(config);
+
+    /* Ignoring coverage since, while it is good to have the null-check, the only current code path always has a default empty array given. */
+    /* istanbul ignore next */
+    if (!params.providers) {
```

Comment:
If this is true then I'd prefer to see a non-null assertion operator used instead of having code that never executes.

Reply:
Ya. I went back and forth on this one. I switched it to the non-null assertion.